scripts/run_example

```diff
@@ -132,4 +132,14 @@ do
   esac
 done
 
+if [[ -n "${SERVER_PID}" ]]; then
```

Comment:
@tiziano88 : it might be a good idea to add a similar check to `runner` &ndash; if the Runtime isn't still running after an example client has finished, that's almost certainly a sign that something's awry.

Reply:
Good call, thanks!